src/Umbraco.Web/Mvc/RenderControllerFactory.cs

```diff
@@ -10,6 +11,10 @@ namespace Umbraco.Web.Mvc
     /// <remarks></remarks>
     public class RenderControllerFactory : UmbracoControllerFactory
     {
+        public RenderControllerFactory(IEnumerable<IRenderController> renderControllers)
```

Comment:
This will be a breaking change, you'll prob have to create an empty obsolete ctor that passes in this collection from the DI Current.Factory